src/components/admin/ConfirmDialog.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import { observer } from 'mobx-react'
+import { Dialog, Card, CardContent, Box, Button, Typography } from '@mui/material'
+import { useTranslation } from 'next-i18next'
+
+import { ModalStore } from 'stores/dashboard/ModalStore'
+
+type Props = {
+  deleteHandler: () => void
+}
+
+export default observer(function ConfirmModal({ deleteHandler }: Props) {
```

Comment:
This file seems to be irrelevant to this PR

Reply:
Removed in https://github.com/podkrepi-bg/frontend/pull/703/commits/e6f154268d2e9a941f0bf9bd92a2bf51f475995f